src/rules/store/prefix-selectors-with-select.ts

```diff
@@ -45,19 +45,18 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
               column: id.typeAnnotation?.range[0] ?? id.range[1],
             },
           },
-          node: id,
```

Comment:
We already have the `loc`, so `node` shouldn't be provided (and I'm curious why it accepts to pass both without complain).